templates/huginn/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Huginn
+description: |
+  Web automation framework. Your agents are standing by!
+version: "0.1"
+category: Automation
+license: MIT
+maintainer: Zicklag <zicklag@katharostech.com>
```

Comment:
Build is still failing, please add maintainer data between quotes.

Reply:
It looks like the build is actually failing because I don't have at least two names before the email address ( I added quotes and the build still fails with the same message ). I can just put my name in there to fix it, but that might be something to update in the tests, unless you really do want to make sure that there are at least a first and a last name in front of the email address.